src/app/components/country-select/country-select.component.html

```diff
@@ -0,0 +1,3 @@
+<select [(ngModel)]="selectedCountry" (change)="onChange($event.target.value)">
```

Comment:
Is there actually any reason for the 2 way biding here? Selected country isn't actually used anywhere?